realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +404,191 @@ public boolean isValid() {
         return this;
     }
 
+    // EqualToAny
+
+    /**
+     * In comparison. This allow you to test if objects matches any value in an array of values.
+     *
+     * @param fieldName the field to compare.
+     * @param values the array of values to compare with.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException if values is empty or do not match class or field type.
+     */
+    public RealmQuery<E> in(String fieldName, String[] values) {
```

Comment:
Since values are given by another method or an existing field, I prefer the array version more than the variant argument version. If we use variant arguments, how can we use another method or an existing array field to fill arguments?